src/main/java/picard/fingerprint/CrosscheckReadGroupFingerprints.java

```diff
@@ -50,7 +50,7 @@
 @CommandLineProgramProperties(
         usage = "Checks if all read groups within a set of BAM files appear to come from the same individual",
```

Comment:
Hmm.  Not my code, but looks like it takes a list of bams as an input and uses them